pkg/corerp/frontend/controller/applications/graph_util.go

```diff
@@ -372,29 +374,21 @@ func applicationGraphResourceFromID(id string) *corerpv20231001preview.Applicati
 		return nil // Invalid resource ID, skip
 	}
 
-	appName := application.Name()
-	appType := application.Type()
-	provisioningState := string(v1.ProvisioningStateSucceeded)
-
-	return &corerpv20231001preview.ApplicationGraphResource{ID: &id,
-		Name:              &appName,
-		Type:              &appType,
-		ProvisioningState: &provisioningState,
+	return &corerpv20231001preview.ApplicationGraphResource{
```

Comment:
I understand that input resource can be version-specific. Ideally, the appgraph's response needs to be version agnostic datamodel. @nithyatsu is there any reason why we use the version-specific response here ?

Reply:
Ideally the app graph API would follow the same versioning principles we have for other APIs 👍 

I think we just didn't think about versioning since it was the first one, but the idea makes sense.